tests/beaker_tests/cisco_interface/test_interface_L3.rb

```diff
@@ -220,13 +220,21 @@ def cleanup(agent, intf, dot1q)
 #################################################################
 test_name "TestCase :: #{tests[:resource_name]}" do
   teardown { cleanup(agent, intf, dot1q) }
-  cleanup(agent, intf, dot1q)
+  # cannot clean up dot1q as most likely not been created yet
+  cleanup(agent, intf)
 
   # -------------------------------------------------------------------
   logger.info("\n#{'-' * 60}\nSection 1. Default Property Testing")
   test_harness_run(tests, :default, harness_class: TestInterfaceL3)
 
-  interface_cleanup(agent, dot1q)
+  # interface_cleanup(agent, dot1q)
```

Comment:
Can we do something like

`  interfaces = get_current_resource_instances(agent, 'cisco_interface')
  interfaces.each do | interface |
    if interface.match(%r#{dot1q}$})
      interface_cleanup(agent, dot1q)
    end
  end`

Reply:
Yea we could do :-)